packages/terra-menu/src/Menu.jsx

```diff
@@ -54,12 +54,17 @@ const propTypes = {
    */
   isArrowDisplayed: PropTypes.bool,
   /**
+   * ![IMPORTANT](https://badgen.net/badge/prop/Important/red)
    * Header Title for main-menu(first-tier).
    * Header Title will only be visible if the main-menu contains at least one sub-menu.
+   * It is expected for the user to pass in the headerTitle as it could possibly be used in cases of responsive
+   * behaviour wherein menu goes bounded/fullscreen.
    */
   headerTitle: PropTypes.string,
   /**
+   * ![IMPORTANT](https://badgen.net/badge/UX/Accessibility/blue)
    * Should the menu display Header Title (first-tier).
```

Comment:
Since we're here, can we also update this?
```suggestion
   * Toggle for the menu display Header Title (first-tier).
```